### PR TITLE
Switch llvm mirror to https://llvm.googlesource.com/

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/build.py [hunk 1]
```diff
@@ -100,9 +100,8 @@
 GITHUB_REMOTE = 'github'
 GITHUB_SSH = 'git@github.com:'
 GIT_MIRROR_BASE = 'https://chromium.googlesource.com/'
-LLVM_OFFICIAL_MIRROR_BASE = GIT_MIRROR_BASE + 'external/llvm.org/'
+LLVM_MIRROR_BASE = 'https://llvm.googlesource.com/'
 GITHUB_MIRROR_BASE = GIT_MIRROR_BASE + 'external/github.com/'
```
Comment (dschuff) [inline src/build.py]: I think we can also remove this line now too.

Comment (sbc100) [inline src/build.py]: Its used in the lines below isn't it?

Comment (dschuff) [inline src/build.py]: oh, right. yeah i was just looking below.

**Comment by jfbastien** [comment]
I love the proliferation of git mirrors.

**Comment by sbc100** [comment]
I'm trying to reduce it .. hopefully we can rid of these ones that I'm deleting here.

**Pull request merged**